README.md

```diff
@@ -111,6 +111,19 @@ For example:
 This can also be used if you just want the channel name to show differently within Slack.
 
 
+#### Ignoring JOIN, PART, QUIT etc
+
+To ignore JOINs, PARTs, QUITs, KICKs and KILLs you can configure your `What I do` JSON object to include,
+
+```json
+{
+  "server": "open.ircnet.net",
+  // ...
+  "mute": ["join", "part", "quit", "kick", "kill"]
```

Comment:
I wonder if passing this into `node-irc` might cause some issues in future versions.

Reply:
Are you thinking like overlapping with options? I could rename it, but it should be fairly short because of the char limit on `What I do` though :/

Reply:
Yeah, now it isn't that straightforward to know which options are used by `slack-irc-client` and which ones are passed down to be used by `node-irc`. Not that it might be that important for the end user though. I probably made an mistake in not namespacing the JSON in the first place, but a bit late for that though.

Maybe if we use something just slightly more verbose for this option. How about `muteIrcEvents` ?

Reply:
Latest commit changed it to `muteIrcEvents` :)